src/panels/lovelace/views/hui-masonry-view.ts

```diff
@@ -260,7 +260,6 @@ export class MasonryView extends LitElement implements LovelaceViewElement {
     return css`
       :host {
         display: block;
-        background: var(--lovelace-background);
```

Comment:
We allow a background per view, so we set the css variable of the right element now?